java/app/src/main/java/com/fullstorydev/shoppedemo/utilities/Constants.java

```diff
@@ -0,0 +1,103 @@
+package com.fullstorydev.shoppedemo.utilities;
+
+import java.util.Calendar;
+
+public class Constants {
+    private static final String[] States={
+        "AL",
+        "AK",
+        "AZ",
+        "AR",
+        "CA",
+        "CO",
+        "CT",
+        "DE",
+        "FL",
+        "GA",
+        "HI",
+        "ID",
+        "IL",
+        "IN",
+        "IA",
+        "KS",
+        "KY",
+        "LA",
+        "ME",
+        "MD",
+        "MA",
+        "MI",
+        "MN",
+        "MS",
+        "MO",
+        "MT",
+        "NE",
+        "NV",
+        "NH",
+        "NJ",
+        "NM",
+        "NY",
+        "NC",
+        "ND",
+        "OH",
+        "OK",
+        "OR",
+        "PA",
+        "RI",
+        "SC",
+        "SD",
+        "TN",
+        "TX",
+        "UT",
+        "VT",
+        "VA",
+        "WA",
+        "WV",
+        "WI",
+        "WY"
+    };
+
+    static public String[] getStates(){ return States; }
+
+    static public boolean isValidState(int state) {
+        if(state < 0 || state > States.length) return false;
+        return true;
+    }
+
+    // generate all valid year selection for credit card expiration year. range: now ~ +10 years
+    static public Integer[] getYears() {
+        int numYears = 10;
+        int currentYear = Calendar.getInstance().get(Calendar.YEAR);
+
+        Integer[] years = new Integer[numYears];
+        for(int i=0;i<numYears;i++){
+            years[i] = currentYear + i;
+        }
+        return years;
+    }
+
+    // generate all valid month selection for credit card expiration year. range: 1~12
+    static public Integer[] getMonths() {
+        int numMonth = 12;
+        Integer[] months = new Integer[12];
+        for(int i=0;i<numMonth;i++){
```

Comment:
NIt: adding whitespace around operators, operands, and semi-colons will makes this more readable"

`for(int i = 0; i < numMonth; i++){`